app/controllers/eligibility_interface/countries_controller.rb

```diff
@@ -6,11 +6,37 @@ def new
       @country_form = CountryForm.new
     end
 
+    def build_url
+      {
+        eligible:
+          Rails
+            .application
+            .routes
+            .url_helpers
+            .eligibility_interface_qualifications_path,
+        ineligible:
+          Rails
+            .application
+            .routes
+            .url_helpers
+            .eligibility_interface_ineligible_path,
+        legacy:
+          Rails
+            .application
+            .routes
+            .url_helpers
+            .eligibility_interface_eligible_path,
+        region:
+          Rails.application.routes.url_helpers.eligibility_interface_region_path
+      }.fetch(eligibility_check.country_eligibility_status)
+    end
```

Comment:
I realise this is what the method was like before, but this might be a nice improvement too!

```suggestion
    def build_url
      url_helpers = Rails.application.routes.url_helpers

      {
        eligible: url_helpers.eligibility_interface_qualifications_path,
        ineligible: url_helpers.eligibility_interface_ineligible_path,
        legacy: url_helpers.eligibility_interface_eligible_path,
        region: url_helpers.eligibility_interface_region_path
      }.fetch(eligibility_check.country_eligibility_status)
    end
```